.github/workflows/sonar-backend.yml

```
@@ -0,0 +1,37 @@
+name: Sonar-Backend
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: Build and analyze
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
+        with:
+          java-version: 21
+          distribution: 'zulu' # Alternative distribution options are available.
```

We don´t want comments in production I guess?
```suggestion
          distribution: 'zulu'
```